apps/api/tests/test_scripts_quality_gates_extended.py

```
@@ -0,0 +1,255 @@
+                    {"name": "CI", "status": "completed", "conclusion": "success"},
+                    {"name": "Coverage 100 Gate", "status": "completed", "conclusion": "success"},
+                ]
+            }
+        return {"statuses": []}
+
+    monkeypatch.setattr(module, "_api_get", fake_api_get)
+
+    _expect(module.main() == 0, "Expected required-checks gate to pass with all contexts successful")
+
+
+def test_check_visual_zero_percy_and_applitools(monkeypatch, tmp_path):
```

Ruff (0.15.2)

[warning] Unused function argument: `tmp_path`

(ARG001)